### PR TITLE
Clean Code - Code order

### PR DESCRIPTION
**Pull request by icottiero**
Task: https://app.productive.io/1-infinum/tasks/4102312
- Added section about the code order
- Added a couple of grammar fixes

### CODE REVIEW

File: Best practices/Clean code.md [hunk 1]
```diff
@@ -59,12 +59,23 @@ The logic contained in a single method should reflect the same level of abstract
 * Methods should not have multiple return params (exception is TryDoSomething pattern which returns bool and resulting object via out param).
 * Avoid using flag arguments. Split the method into several independent methods that can be called from the client without the flag.
 
+### Code order
+
+C# doesn't have any specific requirements for the order of the code inside a class. This is great for us because it gives us the freedom to place code wherever we want, but that doesn't mean that we should just put it anywhere and call it a day.
```
Comment (mahrncic) [inline Best practices/Clean code.md:64]: `... specific requirements for the code order inside a class` maybe? It reads better to me.

File: Best practices/Clean code.md [hunk 2]
```diff
@@ -59,12 +59,23 @@ The logic contained in a single method should reflect the same level of abstract
...
+
+As we mentioned before, the code we write must be understandable to developers as well as the machines. In this context, understandable code must tell a story about the class we are writing, just as if we were writing a newspaper article. First, you get the high-level information, and as you continue reading you dive into more details. Related code should be vertically close, and callers should be above the callees, if possible. Alongside these guidelines, we use the following order:
```
Comment (mahrncic) [inline Best practices/Clean code.md:66]: Nicely put, I like the comparison with the newspaper article 👍

**Review by mahrncic — APPROVED** [review]
Looks good, just added one minor suggestion to potentially improve readability of one sentence.

File: Best practices/Clean code.md (hunk 1 above)
Comment (icottiero) [reply Best practices/Clean code.md]: I agree, changed!

**Pull request merged**